scripts/completion/dvc.bash

```diff
@@ -8,52 +8,41 @@
 #   - https://opensource.com/article/18/3/creating-bash-completion-script
 #----------------------------------------------------------
 
-_dvc_commands='init destroy add import checkout run pull push fetch \
-              status repro remove move unprotect gc config remote metrics \
-              install root lock unlock pipeline commit cache cache_dir diff \
-              config index metrics_add metrics_modify metrics_remove move \
-              metrics_show pipeline_list pipeline_show remote_add version \
-              remote_list remote_default'
+_dvc_commands='init get get_url destroy add remove move unprotect run \
+              repro pull push fetch status gc import import_url config \
+              checkout remote cache metrics install root lock unlock \
+              pipeline daemon commit diff version'
 
 _dvc_options="-h --help -v --version"
-_dvc_global_options="-h --help -q --quiet -V --verbose"
```

Comment:
Hm, `-V` is actually for `dvc --version`. `-v`(small) is for `--verbose`. So the line above needs to be changed as well.

Reply:
ok

Reply:
Thank you! :slightly_smiling_face: